src/MqttPublishing.cpp

```diff
@@ -65,6 +67,27 @@ void MqttPublishingClass::loop()
                 uint16_t maxpower = inv->DevInfo()->getMaxPower();
                 if (maxpower > 0) {
                     MqttSettings.publish(subtopic + "/status/limit_absolute", String(inv->SystemConfigPara()->getLimitPercent() * maxpower / 100));
+
+                    // Publish maxpower and error code to Victron Venus OS
+                    String deviceInstance = MqttSettings.getVictronDeviceInstance(subtopic);
+                    String portalid = MqttSettings.getVictronPortalId();
+                    if ( portalid == NULL ) { portalid = "NOportalId"; }
+
+                    String Vtopic = "W/" + portalid + "/pvinverter/" + deviceInstance + "/ErrorCode";
+                    DynamicJsonDocument val1Doc(128);
+                    val1Doc["value"] = 0;
+                    JsonObject val1Obj = val1Doc.as<JsonObject>();
+                    String data1;
+                    serializeJson(val1Obj, data1);
+                    MqttSettings.publishVictron(Vtopic, data1);
+
+                    Vtopic = "W/" + portalid + "/pvinverter/" + deviceInstance + "/Ac/MaxPower";
+                    DynamicJsonDocument val2Doc(128);
+                    val2Doc["value"] = maxpower;
+                    JsonObject val2Obj = val2Doc.as<JsonObject>();
+                    String data2;
+                    serializeJson(val2Obj, data2);
+                    MqttSettings.publishVictron(Vtopic, data2);
```

Comment:
Why do you have to publish something in the global MqttPublishing class and not moving it to the VictronPublishing module? It would be nice if the victron functionality would be more cental and not distributed via my different locations

Reply:
I hade this 2 publishs first in the victronpublishing part. But it sends every loop the same value without change. So i decided to save resources to move this lines to the maxpower section. So if there are changes, then only one publish for this topic will be sended. 
I can move it back to the Victron part if you want ....

Reply:
Removed all Victron stuff from mqttpublishing.cpp